docs/bugs_and_glitches.md

```diff
@@ -2437,13 +2437,7 @@ This bug can prevent you from talking to Eusine in Celadon City or encountering
 **Fix:** Edit `CheckOwnMonAnywhere` in [engine/pokemon/search_owned.asm](https://github.com/pret/pokecrystal/blob/master/engine/pokemon/search_owned.asm):
 
 ```diff
- 	; If there are no monsters in the party,
- 	; the player must not own any yet.
--; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
- 	ld a, [wPartyCount]
- 	and a
- 	ret z
-
+...
```

Comment:
Why?
```suggestion
...
```

Reply:
I don't have that character/I'm not sure how to type it. Thanks